src/web/components/KeyPairs/KeyPairsTable.tsx

```diff
@@ -34,6 +36,9 @@ function KeyPairsTable({
     setShowKeyPairDialog(false);
   };
 
+  const { sortData } = useSortable<KeyPairModel>();
+  const sortedKeyPairs = keyPairs ? sortData(keyPairs) : [];
```

Comment:
This line got me thinking; since https://github.com/IABTechLab/uid2-self-serve-portal/pull/466/files can't we guarantee that the keyPairs endpoint will return a defined type? If so, we can drop the `| undefined` type from the `keyPairs` prop and remove the check here, then this can just become:
```
const sortedKeyPairs = sortData(keyPairs);
```
or call it `sortedData` like the rest of the changes.

Thoughts?

Reply:
Yes good call!